lib/nylas/draft.rb

```diff
@@ -29,14 +29,34 @@ class Draft
     attribute :unread, :boolean
 
     has_n_of_attribute :events, :event
-    has_n_of_attribute :files, :file
+    has_n_of_attribute :files, :file, read_only: true
+    has_n_of_attribute :file_ids, :string
```

Comment:
We also need to add few tests for this new custom code.

Reply:
Added a new test to check for this and the functionality of extracting `file_ids` from `files`. Let me know if it's good enough!